c/cert/src/rules/ERR33-C/DetectAndHandleStandardLibraryErrors.ql

```diff
@@ -0,0 +1,207 @@
+/**
+ * @id c/cert/detect-and-handle-standard-library-errors
+ * @name ERR33-C: Detect and handle standard library errors
+ * @description Detect and handle standard library errors.
+ * @kind problem
+ * @precision high
+ * @problem.severity error
+ * @tags external/cert/id/err33-c
+ *       correctness
+ *       external/cert/obligation/rule
+ */
+
+import cpp
+import codingstandards.c.cert
+import semmle.code.cpp.commons.NULL
+import codingstandards.cpp.ReadErrorsAndEOF
+import semmle.code.cpp.dataflow.DataFlow
```

Comment:
the `import codingstandards.cpp.ReadErrorsAndEOF` includes `import semmle.code.cpp.dataflow.DataFlow`, although I guess I am not sure on if its better to explicitly import here also, or rm the redundant line